### PR TITLE
Feature#65/style function edit post

### PR DESCRIPTION
**Pull request by germanruzca**
# Description
This is a great change:
- I've added the styles from `scss` to the edit post form.
- Edit and new form for posts a unique one, where there are some conditions to show the respective messages.
- Add the `current_user` method in the controller of the post (`edit`, `create`, `delete`), to do the determinate actions due to the true user that created the post. In the `create` method, it was to save the `user_id` in the post record.
- Add the `stock` and `delivery` values from the params to the permit in the controller and show them in the form view to be seen by the user and fill.
-  A nice effect to show what images previously saved will be purged.
- Add an enum attribute to the delivery field.

For more information, visit issue #65.

newone

- Delete the `create_types` migration and other features derivative.

### CODE REVIEW

File: app/javascript/addItem.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 
+
 window.loadFiles = function(event) {
   const pictures = event.target.files;
+  console.log(pictures);
   for(let i=0; i<pictures.length; i++) {
     images.push(pictures[i]);
   }
+  console.log(images);
+  console.log(event.target.files);
```
Comment (JuanequeX) [inline app/javascript/addItem.js:12]: This `console.logs` are necessary in the code?

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by JuanequeX — APPROVED** [review]
LGTM

File: app/javascript/addItem.js (hunk 1 above)
Comment (germanruzca) [reply app/javascript/addItem.js]: No, a mistake let them in the code. Fixed!

File: app/javascript/addItem.js [hunk 2]
```diff
@@ -1,18 +1,22 @@
 let images = [];
 let container = document.getElementById("upload");
 
+
 window.loadFiles = function(event) {
   const pictures = event.target.files;
+  console.log(pictures);
```
[the remark this answers is not in the document]
Comment (germanruzca) [reply app/javascript/addItem.js]: No, a mistake let it in the code. Fixed, thanks!

**Review by nallely64 — APPROVED** [review]
LGTM!

**Pull request merged**